dashboard/test/controllers/aichat_controller_test.rb

```diff
@@ -97,11 +97,6 @@ class AichatControllerTest < ActionController::TestCase
       ]
     )
 
-    # Note that second expected argument filters out the previous profane message
-    # in what we send to Sagemaker.
-    AichatSagemakerHelper.expects(:format_inputs_for_sagemaker_request).
```

Comment:
Is this covered test case covered elsewhere now?

Reply:
Good call! I'll replace with statement testing that `get_sagemaker_assistant_response` was called with profane messages filtered out. Thanks!